src/iocore/aio/test_AIO.cc

```diff
@@ -108,6 +108,13 @@ struct AIO_Device : public Continuation {
     time_start = 0;
     SET_HANDLER(&AIO_Device::do_hotset);
   }
+  ~AIO_Device()
+  {
+    if (io) {
+      delete io;
+      io = nullptr;
+    }
```

Comment:
It was suggested in the PR/issue call that we could make this a unique_ptr. That's probably worthwhile doing.